openttd.cfg-example

```diff
@@ -0,0 +1,85 @@
+[server]
+# Password required for clients to join
+#server_password = "secretpassword"
+
+# Password for any company if used
+#company_password = "companypass"
+
+# Password for administration console (RCON)
+server_admin_password = "adminpass"
+
+# The port the server listens on for incoming connections (default is 3979)
+server_port = 3979
+
+# Whether or not to advertise this server (true/false)
+server_advertise = true
+
+# How many companies can exist in the game
+max_companies = 8
+
+# How many clients can be connected at once
+max_clients = 50
+
+# If true, the game starts paused, waiting for players
+start_paused = false

```

Comment:
Do you mean 'pause_on_newgame'? Might not be what you want, as then you manually need to unpause.